django/thunderstore/api/urls.py

```diff
@@ -146,4 +147,9 @@
         TeamServiceAccountListAPIView.as_view(),
         name="cyberstorm.team.service-account",
     ),
+    path(
+        "team/<str:team_name>/member/<str:team_member>/update/",
+        UpdateTeamMemberAPIView.as_view(),
+        name="cyberstorm.team.update.member",
```

Comment:
I feel like the name/ID of this view should be `cyberstorm.team.member.update` to match the hierarchy of ownership